Source/EasyNetQ/Extentions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace EasyNetQ
+{
+    public static class Extentions
+    {
+        public static void SafeDispose(this IDisposable disposable)
+        {
+            try
+            {
+                disposable.Dispose();
+            }
+            catch
+            {
+                // Ignore any exceptions
```

Comment:
If we would like to leave this SafeDispose, please add at least a trace `Trace.TraceError(ex.ToString());`